controllers/controller.topics.js

```diff
@@ -0,0 +1,10 @@
+const { getAllTopics } = require('../models/model.topics');
+
+exports.getTopics = (req, res, next) => {
+  const { topics } = req.params;
+  getAllTopics(topics).then((topics) => {
```

Comment:
This is good, except you are giving an argument of `topics` to the model that isn't being used. Line `4` can be removed, there is no parameter on this endpoint